src/modules/sub-items/sub.items.module.js

```diff
@@ -293,72 +447,96 @@ export default class SubItemsModule extends Component {
         this.setState(() => ({ activeView }));
     }
 
-    toggleItemSelect(location, isSelected) {
-        const { selectedLocationsIds } = this.state;
-        const updatedSelectedItemsIds = new Set(selectedLocationsIds);
+    /**
+     * Returns data which we want to cache for selected item
+     * Page data might be discarded but selection of the item should stay,
+     * thus we need item data to perform operations such as bulk delete or bulk move
+     *
+     * @param {Object} item
+     * @param {Object} item.content
+     * @param {Object} item.location
+     *
+     * @returns {Object} dataToCache
+     */
+    getItemDataToCache(item) {
+        const { contentTypesMap } = this.state;
+        const notAvailableLabel = Translator.trans(/*@Desc("N/A")*/ 'content_type.not_available.label', {}, 'sub_items');
+        const contentType = contentTypesMap[item.content.ContentType._href];
+        const contentTypeIdentifier = contentType.identifier;
+        const contentTypeName = window.eZ.adminUiConfig.contentTypeNames[contentTypeIdentifier] || notAvailableLabel;
```

Comment:
If you have `contentTypeIdentifier` it will always have the name in the object `contentTypeNames`. I mean `|| notAvailableLabel` will never be shown. 
The only one what you may not have is content type in `contentTypesMap` but it will throw an exception earlier when you try to read property `identifier`.

Reply:
Fix amended to last commit.